modules/pam_selinux/pam_selinux.c

```diff
@@ -393,7 +393,6 @@ free_module_data(module_data_t *data)
   freecon(data->prev_exec_context);
   if (data->exec_context != data->default_user_context)
     freecon(data->exec_context);
-  memset(data, 0, sizeof(*data));
```

Comment:
This removal is not obvious.

Reply:
The struct `module_data_t` consists of only pointers (so no sensitive data) and the memory is freed on the next line, so there is no further access (e.g. potential double free...).